chat-plugins/dice-commands.js

```diff
@@ -1,185 +1,168 @@
-/* Dice commands chat-plugin
- * by jd
-*/
-var tax = 0.10; // 10%
+/*Dice Plugin by SilverTactic (Silveee)*/
 
-function handleWinnings (bet) {
-	return bet - Math.round(bet * tax);
+'use strict';
+
+if (!global.dicegames) {
+	global.dicegames = {};
+	global.dicegames.count = 0;
+}
+let dicegames = global.dicegames;
+
+const INACTIVE_END_TIME = 60 * 1000; //1 minute
+
+function diceImg(num) {
+	switch (num) {
+		case 0:
+			return "http://i.imgur.com/nUbpLTD.png";
+		case 1:
+			return "http://i.imgur.com/BSt9nfV.png";
+		case 2:
+			return "http://i.imgur.com/eTQMVhY.png";
+		case 3:
+			return "http://i.imgur.com/3Y2hCAJ.png";
+		case 4:
+			return "http://i.imgur.com/KP3Za7O.png";
+		case 5:
+			return "http://i.imgur.com/lvi2ZZe.png";
+	}
```

Comment:
since they're return statements, they aren't needed (no statements after a return statement are executed in a function)